modules/ROOT/pages/writing.adoc

```diff
@@ -671,14 +670,22 @@ For `Overwrite` mode you *must have unique constraints on the keys*.
 
 === Schema optimization operations
 
+
 The Spark Connector supports schema optimization operations via:
 
-* index.
-* constraints.
+* indexes (deprecated);
+* constraints;
+* property type enforcement;
 * set of schema queries.
 
 To speed up the import itself that is executed *before* the import process starts.
 
+
+[NOTE]
+since the version **5.3.0** option **schema.optimization.type** is deprecated in favor of **`schema.optimization.node.keys`**, **`schema.optimization.relationship.keys`** and **schema.type.constraint**.
```

Comment:
[NOTE]
====
Starting from version **5.3.0**, the `schema.optimization.type` option is deprecated in favor of `schema.optimization.node.keys`, `schema.optimization.relationship.keys`, and `schema.type.constraint`.
====

Reply:
What I Have to do about this?

Reply:
Oh I see